vue-components/stories/Dialog.stories.ts

```diff
@@ -23,7 +23,7 @@ export function complex( args: { title: string; actions: string; dismissButton:
 					:title="title"
 					ref="simple"
 					:actions="actions"
-					@action="(_, dialog) => dialog.hide()"
+					@action="function ( _, dialog ){ return dialog.hide() }"
```

Comment:
This should also work without the `return`, right? I don’t think we care about the return value here, only about the effect of calling the function.

Reply:
yep!